introspection/introspection.go

```diff
@@ -232,8 +232,10 @@ func (r *Field) DeprecationReason() *string {
 	if !ok {
 		return nil
 	}
-	reason := args["reason"].(string)
-	return &reason
+	if reason, ok := args["reason"].(string); ok {
+		return &reason
+	}
+	return nil
```

Comment:
Style suggestion. Keep the normal path at lesser indentation, indent the `return nil` paths, like so:

```Go
args, ok := r.field.Directives["deprecated"]
if !ok {
	return nil
}
reason, ok := args["reason"].(string)
if !ok {
	return nil
}
return &reason
```

Reference: https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow.